RomeALincei/ANLcr11.xml

```diff
@@ -0,0 +1,370 @@
+                                <incipit xml:lang="gez">
+                                    ተአምሪሁ፡ ለቅዱስ፡ ጊዮርጊስ። ወኮነ፡ በ፩ እመዋዕል፡ አውዓየ፡ እሳተ፡ ቤቶ፡ ለ፩፡ ብእሲ፡ ወበጎሩ፡
+                                    ሀሎ፡ ፩፡ መነኮስ፡ ወተመይጠ፡ እሳት፡ መንገለ፡ ቤቱ፡ ለውእቱ፡ መነኮስ፡ ወለውለወ፡ ልሳኖ፡ ከመ፡ ያውዒ፡ ቤቶ። ወጸርሐ፡ እንዘ፡ ይብል፡ ኦቅዱስ፡ ጊዮርጊስ፡ ርድአኒ።
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i2.2">
+                                <locus target="#49v"/>
+                                <title ref="LIT6275MGHeathenArmy" type="incomplete"/>
+                                <incipit xml:lang="gez">
+                                    ተአምሪሁ፡ ለቅዱስ፡ ጊዮርጊስ። ወበ፩፡ እመዋል፡ ወፅኡ፡ አረሚ፡ ብሔረ፡ ክርስቲያን፡ ከመ፡ ይፅብኡ።
+                                    ወርእየ፡ ፩፡ መኰንን፡ በሕልሙ፡ ይቤ፡ ዓገቱኒ፡ ሠራዊት፡ ብዙኃን፡ ወ<add
+                                        place="below">ኃ</add>ደጉኒ፡ እለ፡ ምስሌየ፡ ወጐዩ።
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i2.3">
+                                <locus from="49v"  to="50r"/>
+                                <title ref="LIT7356MGHead" type="incomplete"/>
+                                <incipit xml:lang="gez">
+                                    ተአምሪሁ፡ ለቅዱስ፡ ጊዮርጊስ። ወሀሎ፡ ፩፡ ወልድ፡ ዘደወየ፡ ርእሶ፡ በዓቢይ፡ ሕማመ፡ ዓበቅ፡ እስከ፡
+                                    ፀጕረ፡ ቀራንብቲሁ፡ ኮነ፡ ዓበቀ። ወቦአ፡ ውስተ፡ ቤተ፡ ክርስቲያኑ፡ ለቅዱስ፡ ጊዮርጊስ፡ ወተቀብአ፡ እምቅብአ፡ ማኅቶት፡ ዘቤተ፡
+                                    ክርስቲ<pb n="50r"/>ያን፡ ወተፈወሰ፡ ሶቤሃ።
+                                </incipit>
+                            </msItem>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="50r" to="62r"/>
+                            <title ref="LIT1453GadlaG" type="incomplete"/>
+                            <textLang mainLang="gez"/>
+                            <note>Excerpts copied from another manuscript, see <ref target="#e3"/>.</note>
+                            <note>On <locus target="#59v"/>, a letter sent by <persName ref="PRS4333GabraMa"/> to
+                                <persName ref="PRS3429DawitII"/> is mentioned.</note>
+                            <incipit xml:lang="gez">
+                                <locus target="#50r"/>
+                                ንዌጥን፡ <choice><sic>ጺ</sic><corr>ጸ</corr></choice>ሒፈ፡ ዜና፡ ገድል፡ ዘአቡነ፡ ቅዱስ፡ አባ፡ ገብረ፡ መስቀል። 
+                                ወኮነ፡ ስመ፡ አቡሁ፡ ተክለ፡ ኢየሱስ፡ ወስመ፡ እሙ፡ ትርድአነ፡ ማርያም፡ ወሀገረ፡ ሙላዱሰ፡ ምድረ፡ ሰራዌ፡
+                                ዘትሰመይ፡ ጕንደት፡ እስመ፡ እሉ፡ ቅዱሳን፡ አበወ፡ ዝንቱ፡ ቅዱስ፡
+                            </incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i4">
+                            <locus from="62r" to="64r"/>
+                            <title ref="LIT7352HagDosGM" type="incomplete"/>
```

Comment:
Here I do not understand why you place the dossier after gadl?

Reply:
Right, it is a mistake!